hack/set_csv_replaces.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: BUSL-1.1
+
+# Set the `spec.replaces` parameter in the ClusterServiceVersion (CSV) to the
+# previous version, so that the graph of CSVs is built correctly and previous
+# versions show up in OperatorHub.
+#
+# https://access.redhat.com/documentation/en-us/openshift_container_platform/4.2/html/operators/understanding-the-operator-lifecycle-manager-olm
+#
+# Override the `replaces` version by setting PREVIOUS_VERSION to "vX.Y.Z"
+
+PREVIOUS_GIT_TAG=$(git describe --abbrev=0 --tags "$(git rev-list --tags --skip=1 --max-count=1)")
+PREVIOUS_VERSION="${PREVIOUS_VERSION:-${PREVIOUS_GIT_TAG}}"
+CSV="build/bundle/manifests/vault-secrets-operator.clusterserviceversion.yaml"
+
```

Comment:
Suggest failing if `PREVIOUS_VERSION` is empty.

Reply:
👍 Added in 9965b33